proposals/array_modify.json

```diff
@@ -29,15 +29,15 @@
         },
         {
             "name": "index",
-            "description": "The index of the element to insert the value(s) before. If the index is greater than the number of elements, the process throws an `ArrayElementNotAvailable` exception.\n\nTo insert after the last element, there are two options:\n\n1. Use the simpler processes ``array_append()`` to append a single value or ``array_concat()`` to append multiple values.\n2. Specify the number of elements in the array. You can retrieve the number of elements with the process ``count()``, having the parameter `condition` set to `true`.",
+            "description": "The index in the `data` array of the element to insert the value(s) before. If the index is greater than the number of elements in the `data` array, the process throws an `ArrayElementNotAvailable` exception.\n\nTo insert after the last element, there are two options:\n\n1. Use the simpler processes ``array_append()`` to append a single value or ``array_concat()`` to append multiple values.\n2. Specify the number of elements in the array. You can retrieve the number of elements with the process ``count()``, having the parameter `condition` set to `true`.",
             "schema": {
                 "type": "integer",
                 "minimum": 0
             }
         },
         {
             "name": "length",
-            "description": "The number of elements to replace. This parameter has no effect in case the given `index` does not exist in the array given.",
+            "description": "The number of elements in the `data` array to remove (or replace) at the given index. If there are not enough elements to remove, the process simply removes all elements after the given index.",
```

Comment:
"not enough elements to remove" sounds a bit strange to me

proposal:

> If the array contains less elements starting from the given index, the process simply removes up to the end.

Reply:
Isn't "less" wrong here? At least it sounds weird. I've mixed things up and came up with:

> The number of elements in the `data` array to remove (or replace) starting from the given index. If the array contains too few elements to remove, the process simply removes all elements up to the end

Reply:
I mean "less than `length`"

My main issue is just that "too few" or "not enough" sound like there is something wrong with the input (like there are wrongly fewer elements than expected)

Reply:
It reads better if I include the sentence before, indeed. So changed to ~less~ fewer:

>The number of elements in the `data` array to remove (or replace) starting from the given index. If the array contains fewer elements, the process simply removes all elements up to the end.